libminifi/src/core/reporting/SiteToSiteProvenanceReportingTask.cpp

```diff
@@ -53,54 +56,92 @@ void SiteToSiteProvenanceReportingTask::initialize() {
   RemoteProcessorGroupPort::initialize();
 }
 
+void setJsonStr(const std::string& key, const std::string& value, rapidjson::Value& parent, rapidjson::Document::AllocatorType& alloc) { // NOLINT
```

Comment:
why is this here too?

Reply:
It's just a shortcut wrapper for adding members from strings. I ran into a lot of issues with strings being read as garbage. I'll move them to a central location.